packages/web/src/app/Header.tsx

```diff
@@ -50,11 +57,11 @@ export default function Header({ language, onLanguageChange }: HeaderProps) {
                 </DropdownMenuButton>
               ))}
             </DropdownMenuSubmenu>
-            {session.user?.systemRoles.includes(SystemRole.Admin) && (
+            <UserCan action="administer" subject="Language">
```

Comment:
This item showed up for an user that ONLY had the translate role on one language. They had no admin role on any language.